src/pages/getting-started/livechat-apps/webhook-apps/index.mdx

```diff
@@ -94,17 +94,27 @@ The settings page will appear right after a LiveChat user installs your app from
 
 ## 6. Register the webhooks
```

Comment:
@oliwiapolec, could you review changes in this file?

Reply:
all good for me :D